### PR TITLE
 possible fix for #83

### PR DESCRIPTION
**Pull request by bridystone**

### CODE REVIEW

File: ios/Classes/SwiftMapLauncherPlugin.swift [hunk 1]
```diff
@@ -138,11 +138,15 @@ private func showDirections(mapType: MapType, url: String, destinationTitle: Str
 }
 
 
-private func isMapAvailable(map: Map) -> Bool {
-    if map.mapType == MapType.apple {
+private func isMapAvailable(map: Map?) -> Bool {
+    // maptype is not available on iOS
+    if (map == nil) {
+      return false;
```
Comment (mattermoran) [inline ios/Classes/SwiftMapLauncherPlugin.swift:144]: there's usually no need for semicolons in swift. also, more "swifty" way would be like this:
```swift
guard let map = map else {
    return false
}
```

in this case, you don't need to force unwrap `map` below

**Review by mattermoran — CHANGES_REQUESTED** [review]

**Comment by mattermoran** [comment]
If you could update this then I'll merge and release :)

File: ios/Classes/SwiftMapLauncherPlugin.swift (hunk 1 above)
Comment (bridystone) [reply ios/Classes/SwiftMapLauncherPlugin.swift]: it is great what I learn when diving into the whole topics.
Thanks - I more and more like the new programming languages (coming from c/c++ :)

This whole concept of null safety (or whatever it is called in swift) is really great.

BTW: Thanks for the good and well maintained plugin!

Comment (mattermoran) [reply ios/Classes/SwiftMapLauncherPlugin.swift]: Awesome, glad I can be of help :)
Thanks for contributing!

**Review by mattermoran — APPROVED** [review]

**Pull request merged**